pkg/diagnostics/client_test.go

```diff
@@ -0,0 +1,81 @@
+package diagnostics
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/dcos/dcos-core-cli/pkg/pluginutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestClient_CreateErrorsWhenGot503WithStatus(t *testing.T) {
```

Comment:
Is there a convention with httptest where you name the test functions with an underscore, e.g. `TestClient_`?

Reply:
Fixed